pkg/proc/moduledata.go

```diff
@@ -11,9 +11,9 @@ type moduleData struct {
 	typemapVar    *Variable
 }
 
-func (bi *BinaryInfo) loadModuleData(thread *Thread) (err error) {
+func (bi *BinaryInfo) loadModuleData(mem memoryReadWriter) (err error) {
```

Comment:
Would kind of like to take these unexported methods and turn them into just functions that accept `bi *BinaryInfo` ... just seems cleaner to me.